test/functional/tests/failover_standby/test_cache_activation_perf.py

```diff
@@ -0,0 +1,166 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+from datetime import timedelta, datetime
+
+import pytest
+
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.filesystem.file import File
+from test_utils.os_utils import Udev
+from test_utils.size import Size, Unit
+from storage_devices.disk import DiskType, DiskTypeSet
+from storage_devices.device import Device
+from api.cas import casadm, dmesg
+from api.cas.cache_config import CleaningPolicy, CacheMode, CacheLineSize, SeqCutOffPolicy
+
+activation_time_threshold = timedelta(seconds=10)
+test_drive_size = Size(1, Unit.GibiByte)
+dirty_threshold = occupancy_threshold = 99.5
+cls = CacheLineSize.LINE_4KiB
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.nand, DiskType.optane]))
```

Comment:
Current implementation can do with only 1 optane drive. Since you write metadata to a file, you can re-use the same drive for standby instance

Reply:
You're right. For big sizes, it would consume insane amount of temp disk space and time. 
Rewritten to use both disks without temporary file